strongbox-security/strongbox-authentication-registry/src/test/resources/eudst-strongbox-authentication-providers.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:cache="http://www.springframework.org/schema/cache"
+
+       xmlns="http://www.springframework.org/schema/beans"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+                           http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+                           http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache.xsd
+
+                           http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+    <context:annotation-config />
+    <cache:annotation-driven />
+
+    <!-- UserDetailsServices -->
+    <bean id="xmlUserDetailService" class="org.carlspring.strongbox.users.userdetails.XmlUserDetailService"/>
+
+</beans>
```

Comment:
Do we still need this configuration file, now that we're using YAML files for the configuration?

Reply:
yes, we need this. `yaml` won't suit here because this is Spring Application context with security components definition.